anvio/__init__.py

```diff
@@ -520,6 +520,31 @@ def get_args(parser):
              'help': "The taxonomic level to use. The default is '%(default)s'. Only relevant if the\
                       anvi'o ontigs database contains taxonomic annotations."}
                 ),
+    'taxonomy-file': (
+            ['-t', '--taxonomy-file'],
+            {'default': None,
+             'type': str,
+             'help': "Path to The taxonomy file format tsv containe:\
+              ID\td__domaine;p__phylum;[..];s__genus species"}
+                ),
+    'taxonomy-database': (
+            ['-r', '--taxonomy-database'],
+            {'default': None,
+             'type': str,
+             'help': "Path to The directory containe Diamond database for SCGs."}
+                ),
+    'methode': (
```

Comment:
there is no such thing as 'methode'. please go through your code carefully to make sure all variable names are comprehensible, and in proper English as much as possible :)